### ISSUE TITLE
Town Population screen (with the well), the animations of the sprites of the creatures interfere with the consultation of information

### ISSUE THREAD

**Issue by LeHerosInconnu** [ihhub/fheroes2#1787]
Another screen where there shouldn't be the creature sprite animations (https://github.com/ihhub/fheroes2/issues/1772).

In fheroes2, in the Town Population screen, the animations of the sprites of the creatures hinder the consultation of information by hiding the information.
A sprite without animation, as in the original game, is sufficient for this screen intended for the consultation of information.

In original game:

![Town well pop H2 01](https://user-images.githubusercontent.com/43583994/94143063-9736d700-fe6f-11ea-9014-d1df54df2ad8.png)

In fheroes2:

![Town well pop fh2 01](https://user-images.githubusercontent.com/43583994/94143086-9d2cb800-fe6f-11ea-8f20-b06258fa3990.png)

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , this is an intentional change for our project. It feels more alive in comparison with the original game.

**Comment by ihhub** [comment]
As an alternative solution: we can bound all animation within all brown boxes.

**Comment by LeHerosInconnu** [comment]
Hello @ihhub,

> Hi @LeHerosInconnu , this is an intentional change for our project. It feels more alive in comparison with the original game.

I'm ok with that, but provided it doesn't bother the user.
In the case of this screen, clearly this is not suitable.

> As an alternative solution: we can bound all animation within all brown boxes.

You mean cut the part of the sprite out of the brown boxes?
It wouldn't be ideal either.

**Comment by ihhub** [comment]
We could just add then use idle animation where units stands at the same position but show some movements.

**Comment by LeHerosInconnu** [comment]
@ihhub.

> We could just add then use idle animation where units stands at the same position but show some movements.

As long as the animations stay in the frame and therefore do not interfere with the information, and the sprites are not cut, it can be fine.

**Comment by LeHerosInconnu** [comment]
@ihhub.

Another problem with the animation of creatures in the Town Population screen is that opening a creature recruit window stops the animation of creatures in the Town Population screen.
Ideally, it should be ensured that the animations are not stopped.

In fheroes2:

![Stop anim fh2 01](https://user-images.githubusercontent.com/43583994/94260791-10e2c980-ff31-11ea-9c51-8d89c098f0af.png)

**Comment by ihhub** [comment]
Hi @Districh-ru and @zenseii , what do you think about this proposal? Do you prefer to have what we do right now or we do some changes here?

**Comment by zenseii** [comment]
@ihhub, I would prefer to at the least remove the death animation from both the well and the creature info window.
Just the walking and the stopping animation would be perfect for these.

One problem I forsee are dragons and similar that take a lot of horizontal space when flying.

**Comment by Districh-ru** [comment]
@ihhub, I agree with @zenseii and we can bound creatures sprites within light-brown area and possibly also remove some attack (up/down) animations.

**Comment by ihhub** [comment]
Let's do then step-by-step approach: remove death animation from this window.
